README.md

```diff
@@ -24,7 +24,7 @@ sudo dnf install python2-devel python3-devel openldap-devel
 | base_ou                    | yes      |                | Base OU to search for user and group entries                                                                                   |
 | group_dns                  | yes      |                | Which groups user must be member of to be granted access (group names are considered case-insensitive)                         |
 | group_dns_check            | no       | `and`          | What kind of check to perform when validating user group membership (`and` / `or`). When `and` behavior is used, user needs to be part of all the specified groups and when `or` behavior is used, user needs to be part of at least one or more of the specified groups. |
-| host                       | yes      |                | Hostname of the LDAP server                                                                                                    |
+| host                       | yes      |                | Hostname of the LDAP server. Multiple comma-separated entries are allowed.                                                                                                     |
```

Comment:
Nice! :+1: 

We'll need the same for the https://docs.stackstorm.com/authentication.html#ldap, a PR against the
https://github.com/StackStorm/st2docs/blob/master/docs/source/authentication.rst

Reply:
I sent a PR for document update.